# respect users portage optimization flags
--- a/auxil/spicy-plugin/CMakeLists.txt   2022-10-19 18:27:48.807556347 -0500
+++ b/auxil/spicy-plugin/CMakeLists.txt   2022-10-19 18:27:59.724180744 -0500
@@ -74,9 +74,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
-# CMake uses -O2 by default with RelWithDebInfo.
-string(REPLACE "-O2" "-O3" CMAKE_CXX_FLAGS_RELWITHDEBINFO ${CMAKE_CXX_FLAGS_RELWITHDEBINFO})
-
 include_directories(BEFORE ${PROJECT_BINARY_DIR}/include)
 include_directories(BEFORE ${PROJECT_SOURCE_DIR}/include)
 
